app/src/main/res/values/strings.xml

```diff
@@ -99,6 +99,18 @@
     <string name="notes_empty_error">Notes cannot be empty</string>
     <string name="available_to_be_mentor_mentee">Available to be a:</string>
     <string name="settings">Settings</string>
+    <string name="title_activity_about">About Us</string>
+    <string name="about_systers">Systers is an international community for all women involved in the technical aspects of computing. We welcome the participation of women technologists of all ages and at any stage of their studies or careers.
+
+We engage our community by contributing to open source, collaborating with the global community, and learning/enhancing our coding skills. We are committed to providing a safe, positive online community for our many volunteers that offer their skills, time, and commitment to our projects. </string>
+    <string name="about_app">Mentorship System is an application that matches women in tech to mentor each other, on career development, through 1:1 relations during a certain period of time. This is the Android application of this system.</string>
```

Comment:
you can take this part of the text: "This is the Android application of this system."

Reply:
@shiv07tiwari you did not remove this ^^ from the string resource